src/utils/install/journal_defaults.json

```diff
@@ -206,7 +206,7 @@
             "type": "rich-text"
         },
```

Comment:
Should probably use reviewassignment.editor see: https://github.com/BirkbeckCTP/janeway/blob/4ae006429900b643786b6cc19a07b22bc73d3faa/src/review/models.py#L74

Reply:
Ah nice! hadn't noticed the editor was part of that model